boards/hifive1/Makefile.features

```diff
@@ -14,3 +13,5 @@ endif
 
 # The board MPU family (used for grouping by the CI system)
 FEATURES_MCU_GROUP = risc_v
+
+include $(RIOTCPU)/fe310/Makefile.features
```

Comment:
for (all) other boards we use the `-include` notation, does it make sense here to?

Reply:
It's unnecessary to do '-include' as the file exist. And it is my next step to change all to `include`

Reply:
https://github.com/RIOT-OS/RIOT/pull/10078

Reply:
cool, I just opt for consistency when ever possible